expo/ios/ExpoAdapterGoogleSignIn.podspec

```
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '13.0'
+  s.platform       = :ios, '13.4'
```

```suggestion
  s.platform       = :ios
```

how about this? Does that work for you? That way we don't need to worry about the version and we remain backward compatible afaict.